setup.py

```diff
@@ -213,7 +213,7 @@ def run(self):
     'pysha3==0.3',
     'pytest==2.9.2',
     'pytest-cov==2.2.1',
-    'python-bitcoinlib==0.7.0',
+    'python-bitcoinlib', #==0.7.0
```

Comment:
Yeah, I would have preferred "bitcoinlib==0.7.0, ==0.7.1-SNAPSHOT", but pip wasn't liking that. :/ any ideas are welcome.

I don't entirely see it as a bad thing, as I doubt there will be parsing differences between the teo, and TBH, by the time Peter gets around to releasing 0.7.1, we will have an upcoming release of CIP9/10 can can update it to ==0.7.1